pubsub/aws/snssqs/metadata.yaml

```
@@ -0,0 +1,155 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: aws.snssqs
+version: v1
+status: stable
+title: "AWS SNS/SQS"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-aws-snssqs/
+capabilities:
+  - ttl
+builtinAuthenticationProfiles:
+  - name: "aws"
+metadata:
+  - name: region
+    required: true
+    description: |
+      The AWS region to the instance. See this page for valid regions:
+      https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html. 
+      Ensure that DynamoDB are available in that region.
+    example: '"us-east-1"'
+    type: string
+  - name: endpoint
+    required: false
+    description: |
+      AWS endpoint for the component to use. Only used for local development. 
+      The endpoint is unncessary when running against production AWS.
```

So it's the same as the S3 binding

```suggestion
      AWS endpoint for the component to use, to connect to emulators.
      Do not use this when running against production AWS.
```